olly/middleware.py

```diff
@@ -4,15 +4,27 @@
 from profiles.models import UserProfile, BannedUser
 
 
-class CheckBanListMiddleware:
-    def __init__(self, get_response):
-        self.get_response = get_response
+def tenant_middleware(get_response):
```

Comment:
Can we get some documentation on this so people know what and how this works?

Reply:
I plan on removing it as it never worked fully as intended. #34 and #10 will work together as an alternative